cogent/cogent.cabal

```diff
@@ -221,3 +224,30 @@ test-suite test-util
               , filepath >= 1.4.0.0
               , mtl >= 2.2.1
 
+library cogent-quickcheck
```

Comment:
Not sure, why is the cogent compiler a library? I just did it this way because I was basing it off the existing config. Would you prefer I put it all into the config for the test suite?

Reply:
Cogent compiler is a library because the herp people don't want to pull in the whole kit and kaboodle for the small part of the compiler they use.